src/Arbus.Network/INetworkMonitor.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Arbus.Network;
+
+public interface INetworkMonitor
+{
+    bool IsNetworkAvailable { get; }
+    event EventHandler<bool>? NetworkAvailabilityChanged;
+}
+
+[Obsolete($"Use {nameof(INetworkMonitor)}")]
```

Comment:
Why have you decided to keep the old interface?

Reply:
In order to not break code after updating to new version

Reply:
It'll break anyway because we're removing [WindowsNetworkManager.cs](https://github.com/ArbusBiz/Arbus.Network/pull/48/files#diff-9f096f6c9851d88584c4bcdbcf683f48fe357004617009b6870bd91fc34ff8a6)

Reply:
Oh, you're right. I didn't notice it
Should we return old class as obsolute or don't  think about broken code?

Reply:
I believe we do what #47 says.
The reason this PR wasn't closed yet is we're waiting for a moment when we're ready to introduce a breaking change.

Reply:
https://github.com/ArbusBiz/Arbus.Network/pull/55 has breaking change, isn't it?

Reply:
Yup

Reply:
That's why I've started a review here :)